packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/hybrid/adapters.py

```
@@ -13,6 +13,25 @@ def __init__(self, service: VectorSearchService):
+            except Exception:
+                return False
+        return False
+
+    async def supports_qdrant_hybrid(self) -> bool:
+        """Forward the service's pre-flight capability check to the pipeline."""
+        check = getattr(self._service, "supports_qdrant_hybrid", None)
+        if not callable(check):
+            return False
+        try:
+            return bool(await check())
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)